src/test/resources/customer-key-1.json

```diff
@@ -0,0 +1,50 @@
+{
```

Comment:
I would also add customFields section for this json and assert in on the generated draft object. Because transforming customObject has some different verification.

Reply:
I don't think it matters in project-sync. In my code there's no such thing that I care about custom fields. It's a thing for java-sync. I don't want to test java-sync library here :) Here what matters is that it syncs from project to project (which will be tested in a different PR), but that it syncs all attributes or only some of them - this should be tested in java-sync.